test/python/quantum_info/test_synthesis.py

```diff
@@ -105,7 +105,7 @@ def test_euler_angles_1q_random(self, nsamples=100):
         """Verify euler_angles_1q produces correct Euler angles for random unitaries.
         """
         for _ in range(nsamples):
-            unitary = random_unitary(2)
+            unitary = random_unitary(2, seed=442)
```

Comment:
This is just looping n-times over the exact same unitary, rather than sampling a different unitary.
If you wanted to set a deterministic seed for these loops you would need to do something like:
```python
seed = 442
for i in range(nsamples):
    unitary = random_unitary(2, seed=seed+i)
```

Reply:
you are right. Fixed in 052cbbcdc